version: "3.3"

services:

  cappa-antiplag:
    build:
      context: ..
      dockerfile: ./docker/Dockerfile
    container_name: cappa-antiplag
    image: cappa-antiplag
    volumes:
      - ../src:/app/src
    ports:
      - "9020:9020"
    restart: on-failure
    command: gunicorn --pythonpath '/app/src' --bind 0:9020 app.main:app --reload -w 1

networks:
  localhost:
    external: true
